gattrib/src/s_attrib.c

```diff
@@ -94,9 +94,11 @@ char *s_attrib_get_refdes(OBJECT *object)
   OBJECT *slot_text_object;
 
   /*------ Try to get the refdes -----*/
-  temp_uref = o_attrib_search_object_attribs_by_name (object, "refdes", 0);
+  temp_uref =
+    o_attrib_search_object_attribs_by_name (object, (char*) "refdes", 0);
```

Comment:
I feel like the correct thing to do here is to make `o_attrib_search_object_attribs_by_name()` take a `const char*` as its second argument.